handbook/company/communications.md

```diff
@@ -132,16 +69,946 @@ In consideration of our team, Fleet avoids using global tags in channels (i.e. @
       2. Why does this matter? Great question! The Fleet [culture](https://fleetdm.com/handbook/company#culture) is pretty simple: think of others, and remember the company [Values](https://fleetdm.com/handbook/company#values).
 
 
+## Levels of confidentiality
+
+Fleet uses these levels to standardize a commitment to minimal esotericism across the company.  
+- **Public:**   _Share with anyone, anywhere in the world_
+- **Confidential:**  _Share only with team members who've signed an NDA, consulting agreement, or employment agreement_
+- **Classified:**  _Share only with founders of Fleet, business operations, and/or the people involved.  e.g., US social security numbers during hiring_
 
 
-## Levels of confidentiality
+### Legend for Google doc titles
+
+Fleet uses these levels to standardize a commitment to minimal esotericism across the company. 
+- **"public":** _(Available to public, these are rare)_ 
+- **"👀":** _(Confidential - 0p, for Fleet eyes only)_     <!--_(No prefix needed, all Google docs are confidential by default)_-->
+- **"¶👀":** _(E-group - 1p, Direct reports the the CEO)_
```

Comment:
Can you have a read through the diff for typos (specifically in the newly-written sections)?